orders/models.py

```diff
@@ -0,0 +1,56 @@
+from django.db import models
+
+class Cart(models.Model):
+    quantity     = models.IntegerField()
+    user         = models.ForeignKey("users.User",on_delete=models.CASCADE,related_name="cart")
+    product      = models.ForeignKey('products.Product',on_delete=models.CASCADE,related_name='cart')
+    created_at   = models.DateTimeField(auto_now_add=True)
+    updated_at   = models.DateTimeField(auto_now=True)
+    
+    class Meta:
+        db_table = 'carts'
+        
+class Wishlist(models.Model):
+    user         = models.ForeignKey("users.User",on_delete=models.CASCADE,related_name='wishlist')
+    product      = models.ForeignKey('products.Product',on_delete=models.CASCADE,related_name='wishlist')
+    created_at   = models.DateTimeField(auto_now_add=True)
+    updated_at   = models.DateTimeField(auto_now=True)
+    
+    
+    class Meta:
+        db_table = 'wishlists'
+        
+class OrderStatus(models.Model):
+    order_status = models.CharField(max_length=50)
```

Comment:
status로 충분합니다. 나중에 코드에서 `order_status.status`로 해야합니다.
```suggestion
    status = models.CharField(max_length=50)
```

Reply:
수정완료